lib/controllers/controller.js

```diff
@@ -50,6 +50,21 @@ class Controller {
 		return this.fetchChannel(args);
 	}
 
+	patchIdentityFetchChannel(args) {
+		let channelId = (args.req.identity.channel || {}).id;
+
+		if (!channelId && (args.req.identity.audience || []).indexOf('admin') >= 0) {
+			const payload = args.req.body;
+			channelId = payload.channel || payload.id;
```

Comment:
The channel is carried differently while PATCHing an identity vs a regular item.   

Is this an intentional inconsistency?